### PR TITLE
Added missing translations for Polish language

### PR DESCRIPTION
**Pull request by damianszczepanik**

### CODE REVIEW

File: core/src/main/resources/hudson/Messages_pl.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# The MIT License
+#
+# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/Messages_pl.properties]: I doubt this license header is correct

File: core/src/main/resources/hudson/widgets/Messages_pl.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/widgets/Messages_pl.properties]: same

File: core/src/main/resources/hudson/Messages_pl.properties (hunk 1 above)
Comment (damianszczepanik) [inline core/src/main/resources/hudson/Messages_pl.properties]: I've copied this from another properties file

Comment (batmat) [inline core/src/main/resources/hudson/Messages_pl.properties]: Hi, please replace it with 2016 and your own name. Thanks
Le 17 janv. 2016 9:45 PM, "Damian Szczepanik" <notifications@github.com> a
écrit :

Comment (damianszczepanik) [inline core/src/main/resources/hudson/Messages_pl.properties]: I thought you don't like license type. Both files updated

**Pull request merged**